app/views/outcomes/show.html.erb

```diff
@@ -1,4 +1,21 @@
-<%=@outcome.name%>:&nbsp;<%=@outcome.description %>
+<h3><%=@outcome.name%>:&nbsp;<%=@outcome.description %></h3>
```

Comment:
Unlikely. There are a few of these around so I think I'll submit a PR after this one to clean them all up.